src/control/cmd/dmg/storage_test.go

```diff
@@ -71,6 +71,12 @@ func TestStorageCommands(t *testing.T) {
 			printRequest(t, &control.StorageScanReq{NvmeMeta: true}),
 			nil,
 		},
+		{
+			"Scan NVMe health with verbose",
+			"storage scan --nvme-health --verbose",
+			"",
+			errors.New("cannot use --verbose"),
+		},
```

Comment:
Should also test the combination with `--nvme-meta` and without the conflicting options.